docs/guide/core-concepts.rst

```diff
@@ -43,334 +43,399 @@ These definitions are expanded upon in other sections of this document.
+Also, please refer to the `vector documentation <./vectors.rst>`__ for more
+detailed information.
 
 .. raw:: html
 
    <hr>
 
-Tiles
-=====
-
-``Tile`` is a core GeoTrellis primitive. As mentioned in `Tile
-Layers <#tile-layers>`__, a common specification of
-``RDD[(K, V)] with Metadata[M]`` is:
-
-.. code-block:: scala
-
-    type TileLayerRDD[K] = RDD[(K, Tile)] with Metadata[TileLayerMetadata[K]]
-
-What is a ``Tile`` exactly? Below is a diagram of our ``Tile`` type
-hierarchy. As you can see, any ``Tile`` (via ``CellGrid``) is
-effectively a grid of data cells:
-
-.. figure:: ./images/tile-hierarchy.png
-   :alt:
-
-The ``Tile`` trait has operations you'd expect for traversing and
-transforming this grid, like:
-
--  ``map: (Int => Int) => Tile``
--  ``foreach: (Int => Unit) => Unit``
--  ``combine: Tile => ((Int, Int) => Int) => Tile``
--  ``color: ColorMap => Tile``
-
-Critically, a ``Tile`` must know how big it is, and what its underlying
-`Cell Type <#cell-types>`__ is:
+Raster Data
+===========
 
--  ``cols: Int``
--  ``rows: Int``
--  ``cellType: CellType``
+Tiles and Rasters
+-----------------
 
-Fundamentally, the union of a ``Tile`` and ``Extent`` is how GeoTrellis
-defines a ``Raster``:
+The ``geotrellis.raster`` module provides primitive datatypes to represent two
+dimensional, gridded numerical data structures, and the methods to manipulate
+them in a GIS context.  These raster objects resemble sequences of numerical
+sequences like the following (this array of arrays is like a 3x3 tile):
 
-.. code-block:: scala
+.. code::
 
-    case class Raster[+T <: CellGrid](tile: T, extent: Extent) extends CellGrid
+    // not real syntax
+    val myFirstTile = [[1,1,1],[1,2,2],[1,2,3]]
+    /** It probably looks more like your mental model if we stack them up:
+      * [[1,1,1],
+      *  [1,2,2],
+      *  [1,2,3]]
+      */
 
-For performance reasons, we have opted for ``Tile`` to hold its
-``CellType`` as opposed to making ``Tile`` polymorphic on its underlying
-numeric type, for example like ``trait Tile[T]``. The large type
-hierarchy above is what results from this decision. For more
-information, see `our notes on Tile
-performance <../architecture/high-performance-scala.html#the-tile-hierarchy>`__.
+In the ``raster`` module of GeoTrellis, raster data is not represented by
+simple arrays, but rather as subclasses of ``Tile``. That class is more
+powerful than a simple array representation, providing many useful
+operators. Here's an incomplete list of the types of things on offer:
 
-.. raw:: html
+-  Mapping transformations of arbitrary complexity over the constituent
+   cells
+-  Carrying out operations (side-effects) for each cell
+-  Querying a specific tile value
+-  Rescaling, resampling, cropping
 
-   <hr>
+Working with Cell Values
+------------------------
 
-Cell Types
-==========
+Tiles contain numerical data.  These can be of the form of integers, floats,
```

Comment:
`Tiles` should be probably be in-line quoted to match its other occurrences in the file. To in-line quote a word with something that comes directly after (like an `s`) you can do: \``Tile\``\s